recipe/install_nvcc.sh

```diff
@@ -91,18 +91,25 @@ export CMAKE_ARGS="\${CMAKE_ARGS}"
 ### /CMake configurations
 
 mkdir -p "\${CONDA_BUILD_SYSROOT}/lib"
+mkdir -p "\${CONDA_PREFIX}/lib/stubs"
 
 # Add \$(libcuda.so) shared object stub to the compiler sysroot.
 # Needed for things that want to link to \$(libcuda.so).
 # Stub is used to avoid getting driver code linked into binaries.
 
-# Make a backup of \$(libcuda.so) if it exists
-if [[ -f "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so" ]]
+
+if [[ "\${CONDA_BUILD}" == 1 ]]
```

Comment:
Instead of conda-build, do this if `CONDA_BUILD_SYSROOT` is defined

Reply:
Updated to switch when `CONDA_BUILD_SYSROOT` is defined